ROADMAP.md

```diff
@@ -55,6 +55,22 @@ Please file an issue to make suggestions on this roadmap!
       * Dynamic reconfiguration of cluster-autoscaler
       * Automatic discovery of target node pools for cluster-autoscaler
     * Requires much work on CA side
+
+## v0.9.8
+
+  * Experimental support for kube-aws plugins
+  * Tiller installed by default
+    * For use from the plugin support
+
+## v0.9.9
+
+  * RBAC enabled by default
```

Comment:
Accordingly to #655 
My plan is to first ensure all the features to be working with RBAC in v0.9.8.
It would be a matter of just start enabling it by default in v0.9.9 then.